graylog2-server/src/main/java/org/graylog2/indexer/searches/SearchesClusterConfig.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.indexer.searches;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableMap;
+import org.joda.time.Period;
+import org.joda.time.format.ISOPeriodFormat;
+
+import java.util.Map;
+
+@JsonAutoDetect
+@AutoValue
+public abstract class SearchesClusterConfig {
+    private static final org.joda.time.format.PeriodFormatter ISO_PERIOD_FORMAT = ISOPeriodFormat.standard();
+
+    private static final Period DEFAULT_QUERY_TIME_RANGE_LIMIT = Period.ZERO;
+    private static final Map<Period, String> DEFAULT_RELATIVE_TIMERANGE_OPTIONS = ImmutableMap.<Period, String>builder()
+            .put(ISO_PERIOD_FORMAT.parsePeriod("PT5M"), "Search in the last 5 minutes")
```

Comment:
Wouldn't it be easier to use the static factory methods of [`Period`](http://joda-time.sourceforge.net/apidocs/org/joda/time/Period.html), e. g. `Period.minutes(5)`? Or do they result in different objects?